modules/nf-core/last/mafconvert/main.nf

```diff
@@ -4,24 +4,29 @@ process LAST_MAFCONVERT {
 
     conda "${moduleDir}/environment.yml"
     container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container
-        ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/db/db0b5de918238f07ec1ca668be942397da85e26aa582f8927ac37c70896303cf/data'
-        : 'community.wave.seqera.io/library/last:1608--f41c047f7dc37e30'}"
+        ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/37/379183a78f725c3a8f2c4dda2f73ad452e57cc895239938fc97281d7bd74ffbf/data'
+        : 'community.wave.seqera.io/library/last_samtools:e2b51d2d9a1ce9fa'}"
 
     input:
     tuple val(meta), path(maf)
     val(format)
+    path(fasta)
 
     output:
-    tuple val(meta), path("*.axt.gz"),      optional:true, emit: axt_gz
-    tuple val(meta), path("*.blast.gz"),    optional:true, emit: blast_gz
-    tuple val(meta), path("*.blasttab.gz"), optional:true, emit: blasttab_gz
-    tuple val(meta), path("*.chain.gz"),    optional:true, emit: chain_gz
-    tuple val(meta), path("*.gff.gz"),      optional:true, emit: gff_gz
-    tuple val(meta), path("*.html.gz"),     optional:true, emit: html_gz
-    tuple val(meta), path("*.psl.gz"),      optional:true, emit: psl_gz
-    tuple val(meta), path("*.sam.gz"),      optional:true, emit: sam_gz
-    tuple val(meta), path("*.tab.gz"),      optional:true, emit: tab_gz
-    path "versions.yml"                                  , emit: versions
+    tuple val(meta), path("*.axt.gz"),             optional:true, emit: axt_gz
+    tuple val(meta), path("*.bam"),                optional:true, emit: bam
+    tuple val(meta), path("*.blast.gz"),           optional:true, emit: blast_gz
+    tuple val(meta), path("*.blasttab.gz"),        optional:true, emit: blasttab_gz
+    tuple val(meta), path("*.chain.gz"),           optional:true, emit: chain_gz
+    tuple val(meta), path("*.cram"), path(fasta),  optional:true, emit: cram
+    path("*.fai"),                                 optional:true, emit: fai
```

Comment:
```suggestion
    tuple val(meta), path("*.cram"),  optional:true, emit: cram
```
`fasta` is an input file so shouldn't be in the output section,
`fai` should also be an input file :)

Reply:
At the moment I need `path(fasta)` in the `cram` output channel because I did not find another way to access that file during the tests. Can you help me to solve that problem?